src/components/votes.jsx

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { useState } from "react";
+
+const Vote = ({ article }) => {
+  const [toggle, setToggle] = useState(false);
+  const [votes, setVotes] = useState(0);
+  
+
+  const handleClick = () => {
+    setToggle(!toggle);
+    let voteValue = 0;
+    if (toggle) {
+      voteValue = -1;
+    } else {
+      voteValue = 1;
+    }
+    axios
+      .patch(
+        `https://ncnewsyk.herokuapp.com/api/articles/${article.article_id}`,
+        { inc_votes: voteValue }
+      )
+      .then(({ data }) => {
+        setVotes(votes + voteValue);
```

Comment:
You need to optimistically render the votes so that when you click on the button it increases by 1. Currently as you have it in your .then block, it waiting for the patch request to finish until you increment. This means I need to refresh the page to be able to see the changes.